src/cat.jl

```diff
@@ -51,12 +55,13 @@ function ConcatDiskArray(arrays::AbstractArray{<:AbstractArray{T,N},M}) where {T
     startinds = map(first, si)
     sizes = map(last, si)
 
-    return ConcatDiskArray{T,D,typeof(arrays1)}(arrays1, startinds, sizes)
+    chunks = concat_chunksize(D, arrays1)
+    hc = haschunks(first(arrays1))
```

Comment:
Don't the separate concat arrays themselves count as chunks if there are no chunks internally?

Reply:
Yes, that makes more sense. I updated and added a test for this